client/src/Edit/elements/SingleReferenceInput.tsx

```diff
@@ -86,8 +86,9 @@ export default class SingleReferenceInput extends Component<Props, State> {
 
   fetchItems = opts => {
     const { type, model } = this.props;
+
     return api
-      .get(`/${type}/${model}/${query(opts)}`)
+      .get(`/${type}/${model}?${stringify(opts)}`)
```

Comment:
Can stringify return an empty string? I would like to not have dangling ? in these cases.

Reply:
It definitely could!